README.md

```diff
@@ -1,13 +1,400 @@
-wordpress-docker
-============
+# <a name="about"></a>About
 
-Dockerfile source for WordPress [docker](https://docker.io) image.
+This image contains an installation of WordPress 5.x served by an Apache HTTP
+Server on a PHP 7.x runtime.
 
-# Upstream
+For more information, see the
+[Official Image Launcher Page](https://console.cloud.google.com/launcher/details/google/wordpress5-php7-apache).
 
-This source repo was originally copied from:
-https://github.com/docker-library/wordpress
+Pull command (first install [gcloud](https://cloud.google.com/sdk/downloads)):
 
-# Disclaimer
+```shell
+gcloud docker -- pull marketplace.gcr.io/google/wordpress5-php7-apache
+```
 
-This is not an official Google product.
+Dockerfile for this image can be found [here](https://github.com/GoogleCloudPlatform/wordpress-docker/tree/master/5/php7/debian9/apache).
+
+# <a name="table-of-contents"></a>Table of Contents
+* [Using Kubernetes](#using-kubernetes)
+  * [Running Wordpress](#running-wordpress-kubernetes)
+    * [Run Wordpress and MySQL containers](#run-wordpress-and-mysql-containers-kubernetes)
+    * [Run Wordpress connecting to an external MySQL service](#run-wordpress-connecting-to-an-external-mysql-service-kubernetes)
+    * [Run with persistent data volumes](#run-with-persistent-data-volumes-kubernetes)
+* [Using Docker](#using-docker)
+  * [Running Wordpress](#running-wordpress-docker)
+    * [Run Wordpress and MySQL containers](#run-wordpress-and-mysql-containers-docker)
+    * [Run Wordpress connecting to an external MySQL service](#run-wordpress-connecting-to-an-external-mysql-service-docker)
+    * [Run with persistent data volumes](#run-with-persistent-data-volumes-docker)
+  * [Customizing Wordpress](#customizing-wordpress-docker)
+    * [Install additional PHP extensions](#install-additional-php-extensions-docker)
+* [References](#references)
+  * [Ports](#references-ports)
+  * [Environment Variables](#references-environment-variables)
+  * [Volumes](#references-volumes)
+
+# <a name="using-kubernetes"></a>Using Kubernetes
+
+Consult [Launcher container documentation](https://cloud.google.com/launcher/docs/launcher-container)
+for additional information about setting up your Kubernetes environment.
+
+## <a name="running-wordpress-kubernetes"></a>Running Wordpress
+
+This section describes how to spin up a Wordpress service using this image.
+
+### <a name="run-wordpress-and-mysql-containers-kubernetes"></a>Run Wordpress and MySQL containers
```

Comment:
Wordpress -> WordPress here and everywhere

Reply:
Done